src/node/rpc/frontend.h

```diff
@@ -81,11 +81,7 @@ namespace ccf
     kv::Consensus* consensus;
     std::shared_ptr<AbstractForwarder> cmd_forwarder;
     kv::TxHistory* history;
-    // TODO: For now, only the node frontend does not require a valid caller
-    // since nodes only added to the store when they join. This will only be
-    // true when the service is in opening mode.
-    // https://github.com/microsoft/CCF/issues/293
-    bool requires_valid_caller;
```

Comment:
This was just a temporary hack not to check the node caller. Now that the `node_certs` table is gone, this can be remove.